modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -145,6 +145,42 @@ m|+++600+++
 |===
 
 
+== Cloud storage integration settings
+
+Configuration settings are used for custom cloud storage and host authority endpoints, enabling support for non-public and private cloud Azure deployments.
+
+[role=label--enterprise-edition]
+[[config_dbms.integrations.cloud_storage.azb.blob_endpoint_suffix]]
+=== `dbms.integrations.cloud_storage.azb.blob_endpoint_suffix` label:new[Available in 5.26.4]
+
+.dbms.integrations.cloud_storage.azb.blob_endpoint_suffix
+[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
+|===
+|Description
+a|Azure blob storage endpoint suffix.
```

Comment:
I should have added a bit more detail in the code. Something like : 
Azure blob storage endpoint suffix. You will need to change this if you aren't using Azure public cloud (e.g., if you are using Azure Government)